MantidQt/CustomInterfaces/src/Indirect/ConvFit.cpp

```diff
@@ -222,6 +209,72 @@ void ConvFit::setup() {
   updatePlotOptions();
 }
 
+/** Setup FABADA minimizer options
+*
+*/
+void ConvFit::initFABADAOptions() {
```

Comment:
You're right, I hadn't really appreciated that the names will need to be updated anyway and the types make it difficult to work with generically.

Let's leave it as it is.